lambda_function.py

```diff
@@ -18,7 +18,12 @@ def lambda_handler(event, context):
                 omeka_data[text['element']['name']] = text['text']
             if item['files']['count'] > 0:
                 files = requests.get(item['files']['url'])
```

Comment:
The assumption here is that this request is going to succeed 99.99% of the time, but still, it might be worth setting [raise_for_status](https://requests.kennethreitz.org/en/master/api/#requests.Response.raise_for_status) in the unlikely event that it doesn't.

Which also makes me wonder what would happen if the initial request to `http://anth1130.omeka.fas.harvard.edu/api/items/` fails (404, 500, etc). In that situation, would the `omeka.json` still be updated with an empty list of finds?

Reply:
I think you're right. I'll add in raise_for_status where necessary. Thanks for the heads up.